server/src/main/java/org/elasticsearch/index/IndexSettings.java

```diff
@@ -271,6 +261,16 @@ public final class IndexSettings {
         Property.Final,
         Property.IndexScope
     );
+
+    public static final TimeValue DEFAULT_REFRESH_INTERVAL = new TimeValue(1, TimeUnit.SECONDS);
+    public static TimeValue STATELESS_MIN_NON_FAST_REFRESH_INTERVAL = TimeValue.timeValueSeconds(5);
```

Comment:
Since we do no validation against this, can we name it `STATELESS_DEFAULT_REFRESH_INTERVAL` instead?

Also, perhaps add a todo: settle on right interval for stateless.